client/state/sites/media-storage/test/reducer.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_MEDIA_STORAGE_RECEIVE,
+	SITE_MEDIA_STORAGE_REQUEST,
+	SITE_MEDIA_STORAGE_REQUEST_SUCCESS,
+	SITE_MEDIA_STORAGE_REQUEST_FAILURE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import {
+	items,
+	fetchingItems
+} from '../reducer';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'reducer', () => {
+	let sandbox;
+
+	useSandbox( newSandbox => {
+		sandbox = newSandbox;
+	} );
+
+	before( () => {
+		sandbox.stub( console, 'warn' );
+	} );
+
```

Comment:
A test case I'd like to see us include more often in state is to ensure the default reducer export includes the expected keys ([example](https://github.com/Automattic/wp-calypso/blob/f9f0d81e41d9d3f8d33e72772b52ff5239a112af/client/state/ui/editor/test/reducer.js#L13-L20)). Otherwise, it's too easy to forget to add a new reducer function to the default export.